apps/antalmanac/src/components/RightPane/AddedCourses/AddedCoursePane.tsx

```diff
@@ -50,6 +50,7 @@ interface AddedCoursePaneState {
     customEvents: RepeatingCustomEvent[];
     scheduleNames: string[];
     scheduleNote: string;
+    barebonesMode: boolean;
```

Comment:
I like the semantics of `skeleton`, i.e. `displayScheduleSkeleton`

Reply:
I like it. I couldn't come up with anything